tasks/copy_pyqt_to_venv.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -eu
+
+if [[ ! -z "${VIRTUAL_ENV:-}" ]]; then
+    echo -e "copying system's pyqt4 and sip into current python venv ${VIRTUAL_ENV}..."
+    python_version=`python --version 2>&1`
+    package_dir=`find "${VIRTUAL_ENV}/lib" -name "site-packages"`
+    if [[ ${python_version} == *"2.7"* ]]; then
+        cp /usr/lib/python2.7/dist-packages/{sipconfig*.py,sip*.so} "${package_dir}"
+        cp -r /usr/lib/python2.7/dist-packages/PyQt4/ "${package_dir}"
+    elif [[ ${python_version} == *"3"* ]]; then
+        cp /usr/lib/python3/dist-packages/{sipconfig*py,sip*.so} "${VIRTUAL_ENV}/lib/python3.5/site-packages/"
+        cp -r /usr/lib/python3/dist-packages/PyQt4 ${VIRTUAL_ENV}/lib/python3.5/site-packages/
+    else
+        echo "unsupported python version"
+        exit 1
+    fi
+else
+    echo "Not inside a Python virtual env; skipping copying PyQt4 and sip"
+fi
```

Comment:
Does this work on OSX too, or just linux? I spent ages trying to make that work and failed.

Reply:
I don't know, haven't tried. However, the scripts in this PR shouldn't actually use this on macos:
1. travis GUI test on mac is run directly with pytest without tox
2. setup.sh and pyinstaller use pyside for mac as before
So shouldn't be a problem for the moment.

I'll be updating mac to python 3 later on, perhaps after updating Linux first to pyqt5 backend, so eventually we'll probably be using either pyside2 or pyqt5 with python 3.x, and this hack may or may not be needed at that point, to be seen.